municipal_finance/models.py

```diff
@@ -326,7 +335,7 @@ class Meta:
 
 
 class MunicipalityStaffContacts(models.Model):
-    demarcation_code = models.TextField()
+    demarcation_code = models.TextField(primary_key=True)
```

Comment:
watch out - I suspect this is a composite unique key between demarcation_code and role - looking at the comparison on import https://github.com/OpenUpSA/municipal-data/blob/master/sql/upsert/municipality_staff_contacts.sql#L24

and 
```
municipal_finance=> select demarcation_code, role from municipality_staff_contacts order by demarcation_code limit 5;
 demarcation_code |             role             
------------------+------------------------------
 BUF              | Deputy Mayor/Executive Mayor
 BUF              | Municipal Manager
 BUF              | Speaker
 BUF              | Mayor/Executive Mayor
 BUF              | Chief Financial Officer
```

Reply:
Yes. There is a `unique_together` defined in the `Meta` class for the model. I had to add a single primary key to be use the model for querying and creating. Sure you know this but, usually Django models would have an `id` primary key and then unique constraints in these cases. I see in the `sql` folder most of the tables have a `id serial` column defined, but I don't see it in the database. What's up with that? Are those scripts relevant? Can we being adding `id` columns for the models we need to work with?

Reply:
How can this allow the following data:

```csv
demarc,role,name
BUF,Boss,John
BUF,Prole,Peter
```
- doesn't primary key imply unique?

Reply:
I've gone with added an `id` column as mentioned in the previous message.